### PR TITLE
[INTERNAL] Put UI5 Tooling V2 in maintenance

### PR DESCRIPTION
**Pull request by flovogt**
- Remove note informing user about development version
- Put a note that V2 is in maintenance mode

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,8 +8,8 @@
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
 
 > :warning: There are two major versions of UI5 Tooling currently available:
-> - [UI5 Tooling v2](https://sap.github.io/ui5-tooling/stable)- Stable version.
-> - [UI5 Tooling v3](https://sap.github.io/ui5-tooling/next)- Development version.
+> - [UI5 Tooling v2](https://sap.github.io/ui5-tooling/v2)- In maintenance.
```
Comment (ecker) [inline README.md:11]: Better to have v3 listed first. Would just simplify to "Stable Version" and "Maintenance Version".

If you feel cheerful today, put a :tada: after v3, and omit the :warning:

File: overrides/main.html [hunk 2]
```diff
@@ -1,12 +0,0 @@
-{#-
```
Comment (ecker) [inline overrides/main.html:1]: Yeah, lets get rid of this. Maybe later we could introduce a similar hint for the v2 version ...

Comment (flovogt) [reply overrides/main.html:1]: ack

File: README.md (hunk 1 above)
Comment (flovogt) [reply README.md]: done. Great idea!

**Review by RandomByte — APPROVED** [review]

**Review by RandomByte — APPROVED** [review]

**Pull request merged**